asn3/jEdit/org/gjt/sp/jedit/jedit_gui.props

```diff
@@ -883,7 +888,7 @@ about.text=jEdit is brought to you by\n\
 	Ben Williams\n\
 	Bertalan Fodor\n\
 	Bill McMilleon\n\
-	Bj�rn "Vampire" Kautler\n\
+	Bj�rn "Vampire" Kautler\n\

```

Comment:
Hey you got Bjorn to pop up in your CR as well, weird lol

Reply:
can't handle unicode?

Reply:
Classic Eclipse!